### ISSUE TITLE
Side bar text is hidden unless mouse hovered over

### ISSUE THREAD

**Issue by Nicholas-Steel** [Cicero-Interactive/Classic-YouTube-Design#15]
![image](https://user-images.githubusercontent.com/11567846/104276594-effab000-54f0-11eb-95e4-41022c44838c.png)

The image should be pretty self explanatory, but in case it isn't I'm talking about the side bar on the left. Also nice work on resolving the white seams in dark theme.

Firefox 84.0.2
Windows 10 20H2

**Comment by NebelNidas** [comment]
Yeah, this is due to the currently unimplemented dark mode support. It works fine at some places, and I was able to remove the white borders because I'm switching to CSS variables for common stylings, but don't expect proper dark mode support until v0.5. 🤔 On the other hand, this looks like a fast fix, I'll probably implement it on the next release 👍
